lm_eval/evaluator.py

```diff
@@ -57,16 +64,26 @@ def simple_evaluate(
     np.random.seed(1234)
 
     assert tasks != [], "No tasks specified"
-
-    if isinstance(model, str):
-        if model_args is None:
-            model_args = ""
+    assert isinstance(model, str), "Expected the model to be specified as a string"
+    assert isinstance(model_args, str), "Expected model_args to be a string"
```

Comment:
It could be and that would be cleaner, but here I am just trying to make minimal changes to the original script, both to minimize the chance of introducing errors and because there's also some big refactor of `lm-evaluation-harness` [coming soon](https://github.com/EleutherAI/lm-evaluation-harness#language-model-evaluation-harness), so I wanted the fewest number of elements to need to port over, if we ever want to update to the refactor.